demo/src/material-ui/app.js

```diff
@@ -88,7 +96,51 @@ export default class App extends React.Component {
   updateConfig = config => {
     this.setState(config);
   };
+
+  handleFile = element => {
+    // https://www.html5rocks.com/en/tutorials/file/dndfiles/
+    const files = element.files;
+    if (!files.length) {
+      alert("Please select a file!");
+      return;
+    }
+    const file = files[0];
+    const start = 0;
+    const stop = file.size - 1;
+
+    const reader = new FileReader();
+    const that = this;
+    // If we use onloadend, we need to check the readyState.
+    reader.onloadend = function(evt) {
```

Comment:
This will avoid using `that = this`:

```suggestion
    reader.onloadend = evt => {
```

then you could replace any `that` by `this`

Reply:
Amazing .Wow. I like it